Fix docs to make property listing in the website docs more useful. (#5746)
chef/inspec: docs-chef-io/content/inspec/resources/mysql_conf.md

- `('path')` is the non-default path to the `my.cnf` file
- `should eq 'value'` is the value that is expected

## Properties

> This resource supports any of the settings listed in an my.cnf file as properties for e.g. max_connections

Reviewer: ```suggestion
This resource supports any settings listed in a `my.cnf` file as properties. For example, `max_connections`.
```
